teamworkApp/lib/dbCalls.py

```diff
@@ -0,0 +1,109 @@
+# muddersOnRails()
+# Sara McAllister November 5, 2017
+# Last updated: 11-8-2017
+
+# library for SQLite database calls for teamwork analysis app
+
+import contextlib
+import datetime
+import sqlite3
+
+DB = 'db/development.sqlite3'
+test_DB = 'db/test.sqlite3'
+
+def connect(sqlite_file):
+    """ Make connection to an SQLite database file """
+    conn = sqlite3.connect(sqlite_file)
+    c = conn.cursor()
+    return conn, c
+
+def close(conn):
+    """ Commit changes and close connection to the database """
+    conn.commit()
+    conn.close()
+
+@contextlib.contextmanager
+def dbconnect(test):
+    """Create cursor and ensure that DB connection closes"""
+    sqlite_file = DB
+    if test: sqlite_file = test_DB
+
+    conn, cursor = connect(sqlite_file)
+    try:
+        yield cursor
+    finally:
+        close(conn)
+
+def get_all_styles(test=False):
+    """Return all style entries in db ordered based on entry in db"""
+    with dbconnect(test) as c:
+        scores = c.execute('SELECT * FROM styles').fetchall()
+    return scores
+
+def get_students_styles(student_ids, test=False):
```

Comment:
can this work for multiple students' ids? If so, can you clarify your doc string? If not, can you clarify the var name?